sql/plan/showcolumns.go

```diff
@@ -89,14 +89,18 @@ func (s *ShowColumns) Expressions() []sql.Expression {
 	return transform.WrappedColumnDefaults(s.targetSchema)
 }
 
-func (s *ShowColumns) WithExpressions(exprs ...sql.Expression) (sql.Node, error) {
+func (s ShowColumns) WithExpressions(exprs ...sql.Expression) (sql.Node, error) {
```

Comment:
i fix so many bugs from people screwing up value receivers. fixed maybe 2 bugs from misused pointer receivers in the last couple years

Reply:
With* should always use a value receiver, fight me